### PR TITLE
Starter Commit

### LINKED ISSUE #1: Set up a DockerFile

### PR DESCRIPTION
**Pull request by jasgcode**
Makes simple installation of the dependencies

### CODE REVIEW

File: Dockerfile
Comment (calebkrauter) [inline Dockerfile:1]: What is ENV? Also thank you for setting up this Docker!

**Comment by jimothy-dev** [comment]
lgtm

File: run.sh
Comment (calebkrauter) [inline run.sh:1]: What is this file for? What does it do?

**Review by jimothy-dev — APPROVED** [review]
reviewed. looks good

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { createApp } from 'vue'
+import App from './App.vue'
+import router from './router'
+import store from './store'
+
+createApp(App).use(router).use(store).mount('#app')
```
Comment (calebkrauter) [inline src/main.js:6]: What is store?

File: src/index.js
Comment (calebkrauter) [inline src/index.js:1]: Why is this empty?

**Review by calebkrauter — APPROVED** [review]
Nice work, thanks for setting this up!

**Pull request merged**